.bazelversion

```diff
@@ -1 +1 @@
-6.3.1
+7.x
```

Comment:
Shall we keep this pinned

Reply:
If we enable renovate bot we can pin, otherwise I prefer leaving it like this